tests_integration/integration_data_setup.py

```diff
@@ -0,0 +1,168 @@
+from selenium import webdriver
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from contextlib import contextmanager
+import requests
+from requests import ConnectionError
+from urllib.error import URLError
+from os import path
+from time import time, sleep
+import sys
+import subprocess
+from http import HTTPStatus
+
+
+class SetupIridaData:
+
+    def __init__(self, base_url, user, password, branch):
+        """
+        :param base_url: url of the IRIDA instance's API
+        :param user: default admin username
+        :param password: default admin password
+        :param branch: the github branch to run the integration tests on (e.g. 'master' or 'development')
+        """
+
+        self.base_url = base_url
+        self.user = user
+        self.password = password
+        self.driver = None
+        self.branch = branch
+
+        self.IRIDA_PASSWORD_ID = 'password_client'
+        self.IRIDA_AUTH_CODE_ID = 'auth_code_client'
+        self.IRIDA_USER = "admin"
+        self.IRIDA_PASSWORD = "Password1!"  # new password
+
+        self.TIMEOUT = 600  # seconds
+
+        db_name = "irida_uploader_test"
+
+        self.IRIDA_DB_RESET = 'echo '\
+            '"drop database if exists ' + db_name + ';'\
+            'create database ' + db_name + ';'\
+            '"| mysql -u test -ptest'
+
+        root_dir = "tests_integration/tmp"
+        output_files = "tests_integration/tmp/output-files"
+        reference_file = "tests_integration/tmp/reference-files"
+        sequence_files = "tests_integration/tmp/sequence-files"
+
+        self.IRIDA_CMD = ['mvn', 'clean', 'jetty:run',
```

Comment:
Yeah this is the same way it was done in the old uploader. I doubt the integration tests would run on windows at all because of how much they depend on bash.

Reply:
Ok that's fine.  Just curious.  Thanks.